pcweb/pages/docs/hosting/self_hosting.py

```diff
@@ -93,6 +93,14 @@ def self_hosting():
             pc.code("--backend-only"),
             " flags.",
         ),
+        doctext(
+            "It is also possible to export the components without zip it separate. To do this, use the ",
+            pc.code("--no-zip"),
+            " parameter.",
+            "This provides the frontend in the ",
+            pc.code("_static/"),
```

Comment:
Should this be the `.web/_static/` directory?

Reply:
I have add .web/ in front of _static/